src/ext.js

```diff
@@ -530,112 +530,112 @@ export default function ext({ translator, shouldHide, senseNavigation }) {
+                      expression: "optional",
                     },
                   },
                 },
               },
               show: !stylingPanelEnabled,
             },
             icon: {
-              type: 'items',
+              type: "items",
               grouped: true,
-              translation: 'properties.icon',
+              translation: "properties.icon",
               items: {
                 iconSettings: {
-                  type: 'items',
+                  type: "items",
                   items: {
                     useIcon: {
-                      ref: 'style.icon.useIcon',
-                      type: 'boolean',
-                      translation: 'properties.icon.use',
-                      component: 'switch',
+                      ref: "style.icon.useIcon",
+                      type: "boolean",
+                      translation: "properties.icon.use",
+                      component: "switch",
                       options: toggleOptions,
                     },
                     iconType: {
-                      ref: 'style.icon.iconType',
-                      component: 'expression-with-dropdown',
-                      translation: 'properties.icon',
-                      defaultValue: '',
+                      ref: "style.icon.iconType",
+                      component: "expression-with-dropdown",
+                      translation: "properties.icon",
+                      defaultValue: "",
                       options: luiIcons,
-                      expressionType: 'StringExpression',
-                      show: (data) => propertyResolver.getValue(data, 'style.icon.useIcon'),
+                      expressionType: "StringExpression",
+                      show: (data) => propertyResolver.getValue(data, "style.icon.useIcon"),
                     },
                     iconPosition: {
-                      ref: 'style.icon.position',
-                      component: 'dropdown',
-                      translation: 'Common.Position',
+                      ref: "style.icon.position",
+                      component: "dropdown",
+                      translation: "Common.Position",
                       options: [
                         {
-                          translation: 'properties.dock.left',
-                          value: 'left',
+                          translation: "properties.dock.left",
+                          value: "left",
                         },
                         {
-                          translation: 'properties.dock.right',
-                          value: 'right',
+                          translation: "properties.dock.right",
+                          value: "right",
                         },
                       ],
-                      show: (data) => propertyResolver.getValue(data, 'style.icon.useIcon'),
+                      show: (data) => propertyResolver.getValue(data, "style.icon.useIcon"),
```

Comment:
What is up with the `support` declaration at line :651 ?

Also this file seems like a good candiate to follow the max length rule

```js
support: {
      export: !1,
      exportData: !1,
      snapshot: !1,
      viewData: !1,
      quickMobile: !0,
},
```

Reply:
Yeah plenty of other improvements that can be made in this repo. Not sure why this is noted like this, but maybe @Caele had a good reason for this?